source/fixed_array.rb

```diff
@@ -1,3 +1,24 @@
 class FixedArray
+  attr_accessor :size, :array
 
+  def initialize(size, array)
+    @size = size
+    @array = Array.new(size, array)
+  end
+
+  def get(index)
+    if index <= self.size
```

Comment:
`< self.size` — if the size of the array is 3 then you'll have array[0], array[1], and array[2]. array[3] would be invalid.

Add another test to demonstrate that `get` can't access something outside bounds (negative or >= size).